omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -33,7 +33,7 @@
         move "#{install_dir}/etc/datadog-agent/datadog.yaml.example", "/etc/datadog-agent"
         move "#{install_dir}/etc/datadog-agent/trace-agent.conf", "/etc/datadog-agent/trace-agent.conf.example"
         move "#{install_dir}/etc/datadog-agent/process-agent.conf", "/etc/datadog-agent/process-agent.conf.example"
-        move "#{install_dir}/etc/datadog-agent/conf.d", "/etc/datadog-agent"
+        move "#{install_dir}/etc/datadog-agent/conf.d", "/etc/datadog-agent", :force=>true
```

Comment:
in a scenario where you're running the omnibus build twice in the same environment, does this work properly when you add, remove or modify a file in `#{install_dir}/etc/datadog-agent/conf.d` between the 2 builds?

Just want to make the assumption of idempotency is correct :)

Reply:
You'll likely lose the changes you made in `{install_dir}/etc/datadog-agent/conf.d`

Maybe idempotency is a bit too strong, "make sure the build doesn't crash when run twice" is more accurate 😉

Reply:
ok thanks for checking :) just want to make sure we don't expect omnibus to build "clean" packages on "unclean" environments